server/lib/availabilitySync.ts

```diff
@@ -31,21 +31,26 @@ class AvailabilitySync {
     this.radarrServers = settings.radarr.filter((server) => server.syncEnabled);
     this.sonarrServers = settings.sonarr.filter((server) => server.syncEnabled);
 
+    // Initialize and then ping the plex client to
+    // check if it is available before we run the sync
     try {
       await this.initPlexClient();
+      await this.plexClient.getStatus();
+    } catch (ex) {
+      logger.error('Could not connect to your Plex server.', {
+        errorMessage: ex.message,
+        label: 'AvailabilitySync',
+      });
 
-      if (!this.plexClient) {
-        return;
-      }
+      return;
+    }
 
+    try {
```

Comment:
Added a separate try catch to make sure we can access the Plex server. If the call fails, we will return and stop the loop.